Src/EB/AMReX_EB2_3D_C.cpp

```diff
@@ -462,6 +466,9 @@ int build_faces (Box const& bx, Array4<EBCellFlag> const& cell,
 
             if (ncuts > 2) {
                 Gpu::Atomic::Add(dp,1);
+                if (plt_multiple_cuts){
```

Comment:
lets plot the bumber of cuts in order to give more information. So this would look like:

```
            if (ncuts > 2) {
                 Gpu::Atomic::Add(dp,1);
            }
            if (plt_multiple_cuts){
                     mcy(i,j,k) = ncuts;
            }
```

And similar for other places where you set mcx, mcy, mcz

Reply:
dp? or ncuts? dp would be the accumulated number, I think.